cluster-scope/overlays/nerc-ocp-prod/clustersecretstores/nerc-cluster-secrets_patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: external-secrets.io/v1beta1
```

Comment:
For the production cluster,  we're going to want per-namespace secretstores. A cluster-wide secret store would potentially expose sensitive secrets to all cluster users.